.github/workflows/ci.yml

```diff
@@ -1,145 +1,138 @@
-name: CI
```

Comment:
If this file is now auto-generated, we should add a comment at the top about that.